lms/static/coffee/spec/instructor_dashboard/student_admin_spec.coffee

```diff
@@ -0,0 +1,166 @@
+describe 'StudentAdmin', ->
+  studentadmin = {}
+  beforeEach =>
+    loadFixtures 'coffee/fixtures/student_admin.html'
+    # have initialize InstructorDashboard.util.PendingInstructorTasks
+    # since it is user in construct of StudentAdmin
+    window.InstructorDashboard = {}
+    window.InstructorDashboard.util =
+      std_ajax_err: std_ajax_err
+      PendingInstructorTasks: PendingInstructorTasks
+      create_task_list_table: create_task_list_table
+
+    studentadmin = new StudentAdmin $('#student_admin')
+  it 'expects student admin container to be visible', ->
+    expect($('#student_admin')).toBeVisible()
+
+  describe 'EntranceExam', =>
+    it 'Expects student identified input to be visible', =>
+      expect(studentadmin.$field_entrance_exam_student_select_grade).toBeVisible()
+
+    it 'binds to the btn_reset_entrance_exam_attempts on click event', =>
+      expect(studentadmin.$btn_reset_entrance_exam_attempts).toHandle 'click'
```

Comment:
I don't like Jasmine tests that verify that an internal capability has been implemented. It would be better to write a test that verifies that the correct behavior happens when the element is clicked. This goes for a lot of the tests in this file.